src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -650,6 +650,11 @@ void Convolution::setPostOps(dnnl::primitive_attr& attr,
                 if (withSumBroadcast) {
                     break;
                 }
+                eltwisePrecision = fusedEltwisePrecision(fusedWith[i]);
+                if (DnnlExtensionUtils::DataTypeToElementType(outputDataType).size() != eltwisePrecision.size()) {
+                    eltwisePrecision = ov::element::f32;
+                    outputDataType = memory::data_type::f32;
+                }
```

Comment:
this change is to fix some Int8 models(e.g. Denoise/tf/INT8/Denoise.xml) fail with log 'Supported primitive descriptors list is empty for node: Convolution_410 type: Convolution'

for this model conv node(fused with legacy EltwiseAdd postops): 'outputDataType' and 'eltwisePrecision' will be set to f16 by the following code:
https://github.com/openvinotoolkit/openvino/blob/8b1a672c78add030e83b2c2812accf374a2a1fe5/src/plugins/intel_cpu/src/nodes/conv.cpp#L428-L429

so it will not run into the precision alignment logic that you mentioned above(because not match the 'outputDataType != memory::data_type::f16' condition).

Reply:
So both `outputDataType` and `eltwisePrecision` have fp16 precision. Why this is a problem? OneDNN doesn't support such Conv?

Reply:
the `outputDataType` will be set to f32 again:
https://github.com/liubo-intel/openvino/blob/aa9474d446cc9825dc54a1c0b8c826028cb26e39/src/plugins/intel_cpu/src/nodes/conv.cpp#L519

then oneDNN conv_attr_check will fail since `sum_dt=f16, dst_dt=f32 `not match the following condition:
https://github.com/openvinotoolkit/oneDNN/blob/373e65b660c0ba274631cf30c422f10606de1618/src/common/convolution.cpp#L213
  -> https://github.com/openvinotoolkit/oneDNN/blob/373e65b660c0ba274631cf30c422f10606de1618/src/common/primitive_attr.cpp#L448

Reply:
Want it be enough to set eltwisePrecision to fp32 since it is required to aligned with OutputPrecision  https://github.com/liubo-intel/openvino/blob/aa9474d446cc9825dc54a1c0b8c826028cb26e39/src/plugins/intel_cpu/src/nodes/conv.cpp#L518-L520?